src/Components/v2/CollectionsHubsNavItem.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Image, Serif } from "@artsy/palette"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+
+interface CollectionsHubsNavItemProps extends SpaceProps {
+  href: string
+  imageUrl: string
+}
+
+export const CollectionsHubsNavItem: React.FC<CollectionsHubsNavItemProps> = ({
+  href,
+  imageUrl,
+  children,
+}) => {
+  return (
+    <OuterLink to={href}>
+      <ImageContainer>
```

Comment:
This is a lot of layers, but they're needed. 

- The `ImageOverlay` renders a gray overlay on hover of the `OuterLink`. 
- The `ImageContainer` constrains the overlay to be the same size as the image.
- If we use the `OuterLink` to constrain the overlay and ditch the `ImageContainer`, we end up with the overlay covering both the image _and_ the title below it.

Reply:
This makes sense and what I would expect. Thanks for clarifying!

Reply:
👍